mergin/client.py

```diff
@@ -306,6 +306,23 @@ def username(self):
             return None  # not authenticated
         return self._user_info["username"]
 
+    def user_service(self):
+        """
+        Requests information about user from /user/service endpoint if such exists in self.url server.
+
+        Returns response from server as JSON dict or None if endpoint is not found
+        """
+
+        try:
+            response = self.get("/v1/user/service")
+        except ClientError as e:
+            self.log.debug("Unable to query for /user/service endpoint")
```

Comment:
Is this something reported when user submits logs from qgis plugin?
If yes, I suggest to filter out case of 404 response which probably means that such endpoint does not even exists.